gcc/config/arc/arc.h

```diff
@@ -1687,4 +1675,26 @@ enum
 			  && (arc_fpu_build == FPX_QK))
 #define TARGET_FP_ASSIST ((arc_fpu_build & FPX_DP) != 0)
 
+/* Handle pragmas for JLI calls. */
+#define REGISTER_TARGET_PRAGMAS() do {                                  \
+  c_register_pragma (0, "jli_call_fixed",  arc_pr_jli_call_fixed);      \
```

Comment:
I am a bit reluctant in using pragmas with gcc compiler. I am wondering if it is really necessary. I know the MWDT use them quite extensively. But this is not the case of gcc.